types/stripe-js/payment-request.d.ts

```diff
@@ -14,6 +14,11 @@ export interface PaymentRequest {
    */
   show(): void;
 
+  /**
+   * Closes the browser’s payment interface.
```

Comment:
**Suggestion:** Can you add a note in the comment that this is not officially supported by Stripe?

Reply:
Done!